data/spells/scripts/support/levitate.lua

```diff
@@ -0,0 +1,36 @@
+function onCastSpell(creature, variant)
+	local currentPosition = creature:getPosition()
+	local destinationPosition = creature:getPosition()
+	destinationPosition:getNextPosition(creature:getDirection())
```

Comment:
Wait what. It's totally wrong, it should return the next position.

I'll merge this PR and later think about fixing that.